### ISSUE TITLE
Colored output

### ISSUE THREAD

**Issue by KonradHoeffner** [RDFLib/pySHACL#138]
I am currently fixing hundreds of SHACL violations using the default "human" output of  pySHACL, would it be possible to add color to it so it can be parsed by humans more easily?

**Comment by ashleysommer** [comment]
Good idea. I've done coloured outputs for other projects I've worked on. But for some reason, I've never considered to add it for PySHACL. I'll get it implemented before the next release.
